aries-site/src/examples/templates/ascending-navigation/AscendingNavigationAnatomy.js

```diff
@@ -1,85 +0,0 @@
-import React, { useContext } from 'react';
-import {
-  Box,
-  Image,
-  Grid,
```

Comment:
I just want to verify we are good with removing the anatomy? We didnt want to replace it?